packages/volto/locales/volto.pot

```diff
@@ -5192,6 +5299,11 @@ msgstr ""
 msgid "skiplink-navigation"
 msgstr ""
 
+#. Default: "Skip links"
+#: components/theme/SkipLinks/SkipLinks
+msgid "skiplinks"
```

Comment:
```suggestion
msgid "Skip links"
```

Reply:
msgids can be left as lowercased things, in fact when we create them in react they are usually like dictionary keys, so it is quite common to be like that. What is important is to have the correct "Default" value, which in this case is OK.

Reply:
Do we have a standard or style guide defined anywhere? If not, I know @mpalomaki is working on this per https://2025.ploneconf.org/schedule/talks/from-pot-to-perfect-standards-driven-ai-that-gets-plone-volto-ui-translations-right.

For now, I'll edit my suggestions, and resubmit my review.